frontend/src/components/ProjectListView/ProjectListView.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from "react";
+import Button from "@mui/material/Button";
+import { Box, List } from "@mui/material";
+import ProjectView from "../ProjectView/ProjectView";
+import { Project } from "../../types";
+import {
+  createProject,
+  readAllProjects,
+  updateProject,
+  deleteProject,
+} from "../../helpers/projects.helper";
+import { formattedDateString } from "../../utils";
+import "./ProjectListView.css";
+
+const ProjectListView: React.FC = () => {
```

Comment:
Import FC in line 1.

```suggestion
const ProjectListView: FC = () => {
```